src/components/AutocompleteMnxReaction.vue

```diff
@@ -185,12 +199,15 @@ export default Vue.extend({
               )
             )
             .then((response: any) => {
```

Comment:
This definitely needs caching - it is repeatedly requesting full models in each autocomplete

You can probably use `withFullModel`